tasks/main.yml

```diff
@@ -1,5 +1,7 @@
 ---
-# TODO: Consider adding "upgrade=full" to also update everything.
+- include: updates.yml
+  when: with_updates | default('0') == "1"
```

Comment:
Can you add this variable default to `defaults/main.yml`, and also document the variable inside `README.md`? That way we could also just have `when: with_updates`.

Also, can you change the variable name to `update_apt_packages` or something a little more specific, and use `yes` or `no` (bool instead of 0/1).

Or is there some problem passing bools through via Packer?